docs/src/index.md

```diff
@@ -127,6 +129,54 @@ julia> autodiff(Forward, rosenbrock_inp, BatchDuplicated, BatchDuplicated(x, (dx
 (400.0, (var"1" = -800.0, var"2" = 400.0))
 ```
 
+### Convenience functions
+
+!!! note
+    While the convenience functions discussed below use [`autodiff`](@ref) internally, they are generally more limited in their functionality. Beyond that, these convenience functions may also come with performance penalties; especially if one makes a closure of a multi-argument function instead of calling the appropriate multi-argument [`autodiff`](@ref) function directly.
+
+Two convenient functions for common derivative computations are [`gradient`](@ref) and [`jacobian`](@ref).
+As for [`autodiff`](@ref), the mode (forward or reverse) is determined by the first argument.
+
+The function [`gradient`](@ref) computes the gradient of a scalar-valued function that takes in a single vector at a given point.
```

Comment:
For clarity can we say a function with a vector input and scalar return.

Also it may be useful here to mention that these functions take an optional Val(width) where width is how many derivatives to compute in a single batch.

Reply:
However this shouldn't be confused with the Val argument needed for reverse mode jacobian which is the number of returned values

Reply:
I added the clarifications and also discuss the chunk size argument now. Since I believe that, for a complete overview of the interface the user should consult the API docs (this tutorial does not cover all functionality), I also added  a clarifying remark at the beginning of the tutorial that highlights this and links to the API.